src/nvim/ex_docmd.c

```diff
@@ -5310,6 +5333,81 @@ uc_check_code (
     break;
   }
 
+  case ct_MODS:
+  {
+	  int multi_mods = 0;
+	  typedef struct {
+		  int *varp;
+		  char *name;
+	  } mod_entry_T;
+	  static mod_entry_T mod_entries[] = {
+		  {&cmdmod.browse, "browse"},
```

Comment:
@jamessan Where is this `cmdmod` variable defined? I cannot find it anywhere. I can only find `static struct cmdmod` on line 2524 but that doesn't seem to be it.